integration/services/services_test.go

```diff
@@ -25,6 +28,10 @@ func TestServiceInfo(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	imageJSON := string(z.MustBe(json.MarshalIndent(image, "", "    ")))
+
+	fmt.Println("imageJSON:", imageJSON)
```

Comment:
I added this to help debug an integration test failure. Will remove this before merging.